python/ArcadeDrive/robot.py

```diff
@@ -46,6 +46,12 @@ def teleopPeriodic(self):
         self.front_left_motor.set_control(self.left_out.with_output(throttle + wheel))
         self.front_right_motor.set_control(self.right_out.with_output(throttle - wheel))
 
+    def _simulationPeriodic(self):
```

Comment:
robotpy projects are encouraged to use a `physics.py` instead of `_simulationPeriodic`. See https://github.com/robotpy/examples/tree/main/physics